mcsema/BC/Callback.cpp

```diff
@@ -243,7 +243,10 @@ static llvm::Constant *CreateInitializedState(
   // Treat structures as bags of things that can be individually indexed.
   } else if (auto struct_type = llvm::dyn_cast<llvm::StructType>(type)) {
     std::vector<llvm::Constant *> elems;
-    for (const auto field_type : struct_type->elements()) {
+
+    // LLVM 3.5: The StructType::elements() method does not exists!
+    auto struct_type_elements = makeArrayRef(struct_type->element_begin(), struct_type->element_end());
```

Comment:
How exactly does this even work without `llvm::` before the `makeArrayRef`? There shouldn't be any `using namespace llvm;` statements anywhere.